CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
 
+(Note: As a new contributor, please email [Julio Ochoa](JOchoa@esri.com) to join the [Calcite Components GitHub team](https://github.com/orgs/Esri/teams/calcite-components/members). Then, clone the repo via SSH on your machine.)
```

Comment:
Nitpick: 

```md
Note: New contributors should contact ...
```

*no parens + minor tweak

A couple of questions:

* Should this also have a short line about this being needed for the testing setup to work properly?
* How do we communicate this for anyone contributing code that would not get access to the GitHub team?

Reply:
Good questions @jcfranco 👍  - I'll run this by @julio8a and see what he wants to do here.

Reply:
For the long term stability of the repository, we need the PR to come from the repo itself for screener, etc. Would be good to mention this so that folks know why we need to add them, as it is adding a certain barrier to contributing. 

I would add Franco in there too, as a backup.